calico-vpp-agent/cni/network_vpp.go

```diff
@@ -203,6 +203,16 @@ func (s *Server) AddVppInterface(podSpec *storage.LocalPodSpec, doHostSideConf b
 		return swIfIndex, nil
 	}
 
+	// configure MTU from env var if present or calculate it from host mtu
+	var tapMtu int = 0
+	if config.TapMtu > 0 {
```

Comment:
We also need to change the default config.TapMtu value (when `CALICOVPP_TAP_MTU` is not set), in `config.go:80`